src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -58,7 +59,8 @@ public void resetUnjamTimer() {
 
   @Override
   public void periodic() {
-    if (m_revolver.get() > 0.0 && m_revolver.getEncoder().getVelocity() < Constants.Revolver.VELOCITY_THRESHOLD) {
+    if (m_setSpeed > 0.0
+        && (m_revolver.getSelectedSensorVelocity() / 2048d) * 10d * 60d < Constants.Revolver.VELOCITY_THRESHOLD) {
```

Comment:
Move `(m_revolver.getSelectedSensorVelocity() / 2048d) * 10d * 60d` to a public method called `getVelocity` or something like that and call it here

Reply:
Resolved in commit 4b623d0